frontend/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
@alexbridge This file should be removed, because "npm audit" requires a `package-log.json` file in the deployment (or maybe already in the upload process). These problems occurred in the past already.